geopandas/io/file.py

```diff
@@ -241,6 +242,13 @@ def _to_file(
         crs = pyproj.CRS.from_user_input(crs)
     else:
         crs = df.crs
+
+    if driver == "ESRI Shapefile" and any([len(c) > 10 for c in df.columns.tolist()]):
+        warnings.warn(
+            """Column names longer than 10 characters will be truncated when saved to sh
```

Comment:
```suggestion
            """Column names longer than 10 characters will be truncated when saved to 
            shapefile.""", 
            stacklevel=3
```

Reply:
Can you also use `ESRI Shapefile` instead of `shapefile` to be consistent with `driver`?